### ISSUE TITLE
pvc 挂载后pod 使用报错

### ISSUE THREAD

**Issue by HuangTD1** [kubernetes-csi/csi-driver-nfs#605]
**What happened**:
![image](https://github.com/kubernetes-csi/csi-driver-nfs/assets/33741198/e8239953-93be-49a4-9fd4-a394fb623943)

**What you expected to happen**:
导致mariadb 数据无法初始化，报错尝试了好几都不行,但是kubectl exec 进去挂载目录是可读可写的，但是mariadb服务去初始化就不好使nfs server 我给的都是rw 权限
[17:11:59] root:csi-driver-nfs git:(f3a7c1fb*) # cat /etc/exports
/repo *(rw,no_root_squash,no_all_squash,sync)
**How to reproduce it**:

**Anything else we need to know?**:

**Environment**:

- CSI Driver version: 4.6.0
- Kubernetes version (use `kubectl version`): k3s 1.25.7
- OS (e.g. from /etc/os-release): Centos 8.3
- Kernel (e.g. `uname -a`): 5.10.167
- Install tools: nfs
- Others:

**Comment by andyzhangx** [comment]
@HuangTD1 I don't think this is related to this driver since the error msg means the path is also used by other process.

**Comment by andyzhangx** [comment]
@HuangTD1 how did you resolve this issue?
